### PR TITLE
ENH Add micropip.uninstall()

### PR DESCRIPTION
**Pull request by ryanking13**
Related: #51 

Add `micropip.uninstall()` method, which uninstalls packages that are installed.

I used a quite simple approach for this compared to `pip uninstall`, since we only support installing wheels, and wheel metadata contain a list of files in the archive, so I just iterated through the list and removed them.

I didn't add a rollback feature if uninstall fails in this PR. I'm not sure it is worth it, but probably we can add it as a follow up.

- [x] changelog

### CODE REVIEW

**Comment by ryanking13** [comment]
> What would happen if we uninstall a package installed via loadPackage? If it also uninstalls it, we need to be sure loadPackage would be aware of it.

I think removing keys in `loadedPackages` does the job. Will add a test.

**Comment by ryanking13** [comment]
@rth Thanks for the review! I think I addressed all of your comments.

I changed the code a bit after your review (simplified how files are retrieved, and added tests), so I would appreciate it if you can review this one more time.

File: micropip/utils.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from importlib.metadata import Distribution
+from pathlib import Path
+
+
+def get_dist_info(dist: Distribution) -> Path:
```
Comment (rth) [inline micropip/utils.py:5]: Can you either make these functions private (starting with _) or make the whole module private?

**Review by rth — APPROVED** [review]
One minor comment, otherwise LGTM. Thanks!

**Pull request merged**